kitty-items-go/controllers/kibbles.go

```diff
@@ -16,12 +16,12 @@ type kibblesController struct {
 }
 
 type MintKibblesRequest struct {
-	FlowAddress string `json:"flowAddress"`
+	FlowAddress string `json:"flow_address"`
```

Comment:
I think we will probably need to revisit these endpoints, since we've broken compatibility from the JS to Go APIs. Let's merge this one for now